services/sidecar/src/simcore_service_sidecar/cli.py

```diff
@@ -65,7 +60,7 @@ async def run_sidecar(
     )
     try:
         async with DBContextManager() as db_engine, RabbitMQ() as rabbit_mq:
-            next_task_nodes: Optional[List[str]] = await inspect(
+            await inspect(
```

Comment:
maybe the name of the function can be changed to something better now?

Reply:
done